packaging/rpm/Makefile

```
@@ -43,14 +48,15 @@ rpm: srpm
 		-r $(MOCK_CONFIG) \
```

```suggestion
		-r $(MOCK_CONFIG) \
		--bootstrap-chroot \
```

---

I read up a little bit more about the "RHEL7 can't build EL8 RPMs" here: https://github.com/rpm-software-management/mock/wiki/FAQ#i-cannot-build-fedora-or-rhel8-beta-package-on-rhelcentos-7

And this was the solution I found to work.

---

Since we run mock in a fedora:33 container we don't really need --bootstrap-chroot, right?
Or do you mean for the CP centos7 worker?

---

> Since we run mock in a fedora:33 container we don't really need --bootstrap-chroot, right?

Correct.. but...

>  Or do you mean for the CP centos7 worker?

I was avoiding saying that on a public PR, but yes. An alternative suggestion would also be to give us an option override to add other options as we see fit, so in that CP C7 Worker job we can set the parameter. ie:

```suggestion
		-r $(MOCK_CONFIG) \
		ifdef $(MOCK_OPTIONS)
		$(MOCK_OPTIONS) \
		endif
```

And on the CLI run `make MOCK_CONFIG=epel-8-x86_64 MOCK_OPTIONS=--bootstrap-chroot`

---

>  Or do you mean for the CP centos7 worker?

Yes, but lets take another approach with doing options overrides.